src/SeriesSignup/SeriesSignup.vue

```diff
@@ -98,7 +98,7 @@ a {
 }
 .icon__icon {
   stroke: var(--white-100);
-  opacity: .32;
+  opacity: 0.5;
```

Comment:
Where is this value coming from? In Figma it's just 100% opaque. I think it may have been transparent when we did the GoZ page originally, but that was me using rgba white @ 0.32 alpha, because I was lazy.

Apologies if you already know, but in principle, `opacity` should be used instead of `rgba` for icons, to prevent strokes from doubling up like so:

![image](https://user-images.githubusercontent.com/2961244/82053408-38897180-96b5-11ea-819d-c524768177c8.png)

Therefore, this was correct for the original use case. But unless this value is going to be a prop in the component (perhaps it should be if we're enabling theming here), I think we should just keep the opacity at 1 and sacrifice alpha transparency on icons here.